config/daemon/ipv6.md

```diff
@@ -10,30 +10,102 @@ Before you can use IPv6 in Docker containers or swarm services, you need to
 enable IPv6 support in the Docker daemon. Afterward, you can choose to use
 either IPv4 or IPv6 (or both) with any container, service, or network.
 
-> **Note**: IPv6 networking is only supported on Docker daemons running on Linux
-> hosts.
+> **Note**
+>
+> - IPv6 support is experimental, use it with caution.
```

Comment:
I think the experimental only applies to NAT and port-mapping with IPv6, so wondering if we need to clarify that.

Reply:
`ip6tables` does a bit more than enabling NAT-based port-mapping, as described [here](https://github.com/docker/docs/pull/17176#discussion_r1211551672). And overall, IPv6 support is half baked, so I think it's good to keep this.